Rasa_Bot/custom_channels.py

```diff
@@ -0,0 +1,70 @@
+import inspect
+from typing import Text, Callable, Awaitable, Any
+
+from rasa.core.channels.channel import InputChannel, OutputChannel, UserMessage
+from sanic import Blueprint, response
+from sanic.request import Request
+from sanic.response import HTTPResponse
+
+from niceday_client.niceday_client import NicedayClient
+
+
+class NicedayOutputChannel(OutputChannel):
+    """
+    Output channel that sends messages to Niceday server
+    """
+    def __init__(self):
+        self.niceday_client = NicedayClient()
+
+    def name(self) -> Text:
+        return "niceday_output_channel"
+
+    async def send_text_message(
+        self, recipient_id: Text, text: Text, **kwargs: Any
+    ) -> None:
+        """Send a message through this channel."""
+        for message_part in text.strip().split("\n\n"):
```

Comment:
Good point, but I think it will be fine. I copy-pasted this from https://github.com/RasaHQ/rasa/blob/8815a135e9c795dd6545faf881d8da22b798603a/rasa/core/channels/slack.py#L55 .